p2p/libp2p/netMessenger.go

```diff
@@ -718,7 +718,7 @@ func (netMes *networkMessenger) BroadcastOnChannelBlocking(channel string, topic
 
 func (netMes *networkMessenger) checkSendableData(buff []byte) error {
 	if len(buff) > maxSendBuffSize {
-		return p2p.ErrMessageTooLarge
+		return fmt.Errorf("%w, maximum: %d, received %d", p2p.ErrMessageTooLarge, maxSendBuffSize, len(buff))
```

Comment:
maybe switch order? received and then maximum? also, you can specify the unit (number of bytes). same above

Reply:
also fix failing tests (replace with `errors.Is` and maybe add a `strings.Contains()` for extra check)

Reply:
ok, this PR will close, will take into account your comment :)